### PR TITLE
[WIP] Disable KeepAlives for HA tests

### PR DESCRIPTION
**Pull request by mgencur**
Trying to see if it affects errors in the HA tests.

### CODE REVIEW

File: test/ha/ha.go [hunk 1]
```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package ha
 
 import (
+	"knative.dev/pkg/test/spoof"
```
Comment (mattmoor) [inline test/ha/ha.go:20]: Format Go code:
```suggestion

```

File: test/ha/ha.go [hunk 2]
```diff
@@ -17,6 +17,8 @@ limitations under the License.
...
+	"net/http"
 	"testing"
```
Comment (mattmoor) [inline test/ha/ha.go:22]: Format Go code:
```suggestion
	"testing"

	"knative.dev/pkg/test/spoof"
```

**Comment by mgencur** [comment]
OK. The latest failure shows that disabled KeepAlives don't make a difference.

**Pull request closed without merging**